src/reuse/modules/service/odata.ts

```diff
@@ -350,7 +350,11 @@ export class OData {
     return new Promise((resolve, reject) => {
       this.curl.get(url, options, function (error: any, res: any, body: any) {
         if (!error) {
-          resolve(body);
+          if(res.statusCode >= 400) {
```

Comment:
please use a formatter
```suggestion
          if (res.statusCode >= 400) {
```